daemon/SecretsImpl/secretscryptohelpers.cpp

```diff
@@ -431,6 +458,11 @@ Daemon::ApiImpl::SecretsRequestQueue::asynchronousCryptoRequestCompleted(
             emit userInputCompleted(cryptoRequestId, result, input);
             break;
         }
+        case QueryLockStatusCryptoApiHelperRequest: {
+            bool locked = parameters.size() ? parameters.first().value<bool>() : true;
```

Comment:
I'd have preferred !parameters.isEmpty() or parameters.size() > 0 in favour of implicitly casting size() to bool. This works as well. Just a note.